pyproject.toml

```diff
@@ -87,7 +87,7 @@ shapely = { version = ">=2,<3", optional = true }
 # also, we don't support arbitrarily old versions of this library
 snowflake-connector-python = { version = ">=3.0.2,<4", optional = true, extras = [
   "pandas",
-] }
+], allow-prereleases = true }
```

Comment:
I plan to remove this once the release is out. I'll move this out of draft mode then as well.